tests/CMakeLists.txt

```diff
@@ -131,6 +131,17 @@ add_test_case(websocket_midchannel_write_huge_message)
 #add_test_case(websocket_midchannel_no_more_shutdown_callback) ???
 add_test_case(websocket_midchannel_read_message)
 add_test_case(websocket_midchannel_read_multiple_messages)
+add_test_case(websocket_boot_sanity_check)
+add_test_case(websocket_boot_golden_path)
+add_test_case(websocket_boot_fail_at_http_connect)
+add_test_case(websocket_boot_fail_at_http_connect_error)
+add_test_case(websocket_boot_fail_at_new_request)
+add_test_case(websocket_boot_fail_at_response_error)
+add_test_case(websocket_boot_fail_at_response_status)
+#add_test_case(websocket_boot_fail_at_response_handshake_key)
```

Comment:
actually, what's the deal here?  maybe delete if no longer applicable

Reply:
It's a test I need to write when I finish implementing the full websocket spec. I'll remove this. I have another TODO in the code about it